_config.yml

```diff
@@ -47,3 +49,12 @@ defaults:
       path: ""  ## all pages
     values:
       image: https://bitcoinops.org/img/logos/optech-notext.png
+  - scope:
+      path: "_topics"
```

Comment:
Good point.  I'm moving the files to `_topics/en/` but not otherwise adding any localization features (e.g., if someone adds `_topics/es/foo.md`, we'll need to tweak the templates to ignore that entry on the English pages).